1-js/05-data-types/05-array-methods/article.md

```diff
@@ -767,7 +767,7 @@ A cheat sheet of array methods:
 - Additionally:
   - `Array.isArray(value)` checks `value` for being an array, if so returns `true`, otherwise `false`.
 
-Please note that methods `sort`, `reverse` and `splice` modify the array itself.
+Please note that methods `push`, `pop`, `shift`, `unshift`, `sort`, `reverse` and `splice` modify the array itself.
```

Comment:
don't think these are needed

Reply:
I agree. This addition would be distracting. The text is a warning for methods that modify the original array as a maybe unwanted side effect.